dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/tests/ejb/AsyncEJBTest.java

```diff
@@ -53,6 +54,7 @@ public static void setUp() throws Exception {
                         .addAsManifestResource(AsyncEJBServlet.class.getPackage(), "permissions.xml", "permissions.xml");
         ShrinkHelper.exportAppToServer(server, app, DeployOptions.SERVER_ONLY);
         server.startServer();
+        assertNotNull("Security service did not start", server.waitForStringInLog("CWWKS0008I"));
```

Comment:
Might be useful to have a comment which says what `CWWKS0008I` really is in English?

Reply:
Yeah, I should have put that in.

It's not too bad in this case since it's `CWWKS0008I: The security service is ready.` so the assertion message does tell you everything you need to know but I'll go and add it now.

Reply:
done